release/tune_tests/cloud_tests/workloads/_tune_script.py

```diff
@@ -39,35 +51,57 @@ def on_step_begin(self, iteration, trials, **info):
             fp.write("1")
 
 
-def run_tune(
-        no_syncer: bool,
-        upload_dir: Optional[str] = None,
-        experiment_name: str = "cloud_test",
-        indicator_file: str = "/tmp/tune_cloud_indicator",
-):
-    num_cpus_per_trial = int(os.environ.get("TUNE_NUM_CPUS_PER_TRIAL", "2"))
+def run_tune(no_syncer: bool,
+             upload_dir: Optional[str] = None,
+             experiment_name: str = "cloud_test",
+             indicator_file: str = "/tmp/tune_cloud_indicator",
+             trainable: str = "function",
+             num_cpus_per_trial: int = 2):
+    if trainable == "function":
+        train = fn_trainable
+        config = {
+            "max_iterations": 30,
+            "sleep_time": 5,
+            "checkpoint_freq": 2,
+            "score_multiplied": tune.randint(0, 100),
+        }
+        kwargs = {"resources_per_trial": {"cpu": num_cpus_per_trial}}
+    elif trainable == "rllib_str" or trainable == "rllib_trainer":
+        if trainable == "rllib_str":
+            train = "PPO"
+        else:
+            train = PPOTrainer
+
```

Comment:
This is just expanding the test right? Did we make sure that the release test passed before merging? (Just a question wrt approving the patch to go onto the 1.9 release)

Reply:
Yes, I ran the existing release tests manually on the cluster to make sure they still work (unfortunately no buildkite logs)

The release test failures in the current master build are due to bookkeeping and I'll investigate them today.